app/services/metrics/daily/review_turnaround/per_user_project.rb

```diff
@@ -0,0 +1,80 @@
+module Metrics
+  module Daily
```

Comment:
Being this daily, shouldn't the interval set to `Time.zone.now.all_day`? Or it could be any other day?

Reply:
uhmm i think yes it could be, but i didn't change it thinking about the manually metric process.. like when we want to process an specific day